.github/workflows/zig.yaml

```diff
@@ -130,13 +133,23 @@ jobs:
         if: matrix.settings.host == 'ubuntu-latest'
       - name: Test
         uses: addnab/docker-run-action@v3
-        if: matrix.settings.target == 'aarch64-unknown-linux-musl'
+        if: matrix.settings.host == 'ubuntu-latest' && matrix.settings.target != 'x86_64-unknown-linux-musl' && matrix.settings.target != 'armv7-unknown-linux-musleabihf'
         with:
           image: node:lts-alpine
-          options: --platform linux/arm64 -v ${{ github.workspace }}:/build -w /build
+          options: --platform linux/${{ matrix.settings.docker-platform }} -v ${{ github.workspace }}:/build -w /build
           run: |
             set -e
             yarn test
+      - name: Test
+        uses: addnab/docker-run-action@v3
+        if: matrix.settings.target == 'armv7-unknown-linux-musleabihf'
+        with:
+          # https://github.com/nodejs/docker-node/issues/1798
+          image: node:18-alpine
+          options: --platform linux/${{ matrix.settings.docker-platform }} -v ${{ github.workspace }}:/build -w /build
+          run: |
+            set -e
+            yarn test:without-lerna
```

Comment:
lerna (nx) doesn't support armv7 so I had to run it directly with yarn. I used node 18 for this test because when using node 20 `yarn test` got stuck (https://github.com/nodejs/docker-node/issues/1798).